GreysonElkins/exquisite-corpse
# Routing/cc setup router

#### What's this PR do?
* Sets up router 
* Makes directories and files for the components

#### Where should the reviewer start?
* They can start in the component directory where all the new directories and files live 
* index.js

#### How should this be manually tested?
* Open up the browser with `npm start` to see that everything still displays, but it's not needed

#### Any background context you want to provide?
* Any route names can be changed if necessary 

#### What are the relevant tickets?
Closes #21 

<iframe src="https://giphy.com/embed/cfuL5gqFDreXxkWQ4o" width="480" height="480" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/cat-cool-cats-cfuL5gqFDreXxkWQ4o">via GIPHY</a></p>
